rasa_core/featurizers.py

```diff
@@ -89,17 +89,25 @@ def prepare_from_domain(self, domain):
                                  len(domain.entity_states))
         self.slot_feature_len = len(domain.slot_states)
 
-    def encode(self, states):
+    def encode(self, state):
         # type: (Optional[Text, float]) -> np.ndarray
         """Returns a binary vector indicating which features are active.
 
-        Given a dictionary of states (e.g. 'intent_greet',
-        'prev_action_listen',...) return a binary vector indicating which
-        features of `self.input_features` are in the bag. NB it's a
-        regular double precision float array type.
+        If TrackerFeaturizer.binary_intent_prob is True (default behaviour):
```

Comment:
Am I missing something or is this function not looking at `binary_intent_prob`? I mean yes, the input will be different depending on that parameter, but I think this description is not at the right location.

Reply:
You're right, I wanted to emphasize here that former `Binary-` and `Probabilistic-` is the same thing now. I'll need to rephrase it